clustersmgmt/v1/listening_method_type.go

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright (c) 2019 Red Hat, Inc.
```

Comment:
2020 :)

Reply:
again, it's generated, the scripts need an update :)